resources/scripts/frontend/index.js

```diff
@@ -6,5 +6,7 @@ import '@images/favicon.ico';
 import 'airbnb-browser-shims';
 // Uncomment the following line to enable lazy-loading of media images:
 // import './lazyload';
+// Uncomment the following line to enable sprite svg:
```

Comment:
If there are no svgs used and a sprite is not generated then we can enable it by default as it has not cost.
```suggestion
```